x/concentrated-liquidity/lp.go

```diff
@@ -335,13 +335,16 @@ func (k Keeper) initializeInitialPositionForPool(ctx sdk.Context, pool types.Con
 
 	// Calculate the spot price and sqrt price from the amount provided
 	initialSpotPrice := amount1Desired.ToDec().Quo(amount0Desired.ToDec())
-	initialSqrtPrice, err := initialSpotPrice.ApproxSqrt()
+
+	// Calculate the initial tick from the initial spot price
+	initialTick, err := math.PriceToTickRoundBankers(initialSpotPrice, pool.GetTickSpacing())
```

Comment:
Would be nice to have an in line comment here explaining why we use bankers rounding for future reviewers